src/core/style/_mixins.scss

```diff
@@ -39,6 +39,18 @@
 }
 
 /**
+ * This mixin overrides default button styles like the gray background,
+ * the border, and the outline.
+ */
+@mixin md-button-reset {
```

Comment:
I kind of want to stop putting stuff into `mixin.scss` since it's too much of a random grab-bag. Can we add a `button-mixins` file?